packages/core/src/dialects/sqlite/query-generator-typescript.ts

```diff
@@ -15,6 +15,33 @@ export class SqliteQueryGeneratorTypeScript extends MySqlQueryGenerator {
     return `PRAGMA TABLE_INFO(${this.quoteTable(tableName)});`;
   }
 
+  describeCreateTableQuery(tableName: TableNameOrModel) {
+    const table = this.extractTableDetails(tableName);
+    let tableAndSchema;
+    if (!table.schema || table.schema === this.dialect.getDefaultSchema()) {
+      tableAndSchema = table.tableName;
+    } else {
+      tableAndSchema = `${table.schema}${table.delimiter || '.'}${table.tableName}`;
+    }
```

Comment:
This code snippet could be moved to a private sqlite method called `escapeTable`

Reply:
Done